tsl/src/reorder.c

```diff
@@ -276,6 +276,9 @@ tsl_subscription_exec(PG_FUNCTION_ARGS)
 	List *parsetree_list;
 	ListCell *parsetree_item;
 
+	if (!subscription_cmd)
+		PG_RETURN_VOID();
```

Comment:
I suggest to error out in such case. I believe this function is for internal use only (used by copy/move chunk), ideally it should not accept NULL values

Reply:
I disagree we don't error on empty string so not erroring on NULL would be consistent with that.